pkg/util/system/cgroup_resource.go

```diff
@@ -41,7 +41,8 @@ const (
 	CPUSFileName      = "cpuset.cpus"
 	CPUTaskFileName   = "tasks"
 
-	CpuacctStatFileName = "cpuacct.stat"
+	CpuacctStatFileName  = "cpuacct.stat"
```

Comment:
is `CpuacctStatFileName` still being used?

Reply:
I'm afraid so, because `collectNodeResUsed` read node cpu usage from `/proc/stat`,  which still use USER_HZ as unit.

Reply:
it's useless now.

Reply:
fixed